[Design] 스타일 변수 값 변경에 따른 수정 및 리뷰, 장소카드 상세페이지 이동 기능 추가

soohyn opened the pull request:
## PR 유형
- [] 새로운 기능 추가
- [] 버그 수정
- [X] CSS 등 사용자 UI 디자인 변경
- [X] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [] 코드 리팩토링
- [] 주석 추가 및 수정
- [] 문서 수정
- [] 테스트 추가, 테스트 리팩토링
- [] 빌드 부분 혹은 패키지 매니저 수정
- [] 파일 혹은 폴더명 수정
- [] 파일 혹은 폴더 삭제

## PR 체크리스트
<!-- PR이 다음 요구 사항을 충족하는지 확인하세요. -->
- [X] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [X] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

### PR 상세
- text-xs → paragraph-xs 로 변경
- 리뷰카드 디자인 의논 후 글자 크기 결정 및 변경
- 리뷰, 장소 컴폰너트 상세페이지 이동 기능 추가

## 이슈
resolves #27 #26

soohyn committed:
[Design] imageField 컴포넌트 스타일 오류 수정

- 오타 제거

Resolves 여백 변수 변경에 따른 스타일 점검 및 수정 #27

soohyn committed:
[Design] 스타일 변수 수정에 따른 스타일 값 수정

- padding 값 전체 점검 후 수정

Resolves 여백 변수 변경에 따른 스타일 점검 및 수정 #27

juhyuny commented on src/components/molecules/PlaceCard.tsx:
{address}부분 text-xs  ->  paragraph-xs 수정부탁드림다!

soohyn replied:
감삼당 수정하고 푸시 바로 할게여

soohyn replied:
텍스트는 생각 못했는데 address 말고도 많아서 다 수정하고 올릴게여

soohyn committed:
[Design] text style 변수 변경에 따른 수정

- text-xs -> paragraph-xs

Resolves 여백 변수 변경에 따른 스타일 점검 및 수정 #27

soohyn committed:
[Refact] review card

- 텍스트 스타일 수정 (변수 변경에 따른 수정)
- more 버튼 추가
- Link 태그 추가: 상세페이지 이동 기능

장소 카드, 리뷰 카드 상세 페이지 이동 기능 추가 #26

soohyn committed:
[Refact] place card

- Link 태그 추가: 상세페이지 이동 기능
- 북마크 버튼 컴포넌트 분리

Resolves 장소 카드, 리뷰 카드 상세 페이지 이동 기능 추가 #26

soohyn committed:
[Refact] navlink isActive 기능 활용

- isCurrentPath 변수 제거 - NavLink classname 내장 함수 활용

Resolves 장소 카드, 리뷰 카드 상세 페이지 이동 기능 추가 #26

soohyn committed:
[Refact] TabNavItem 수정

- button 에서 Link태그로 수정

Resolves 장소 카드, 리뷰 카드 상세 페이지 이동 기능 추가 #26

soohyn committed:
[Rename] ReviewCard molecules로 이동

Resolves 장소 카드, 리뷰 카드 상세 페이지 이동 기능 추가 #26

juhyuny commented on src/components/molecules/PlaceCard.tsx:
컴포넌트 제목으로는 h2가 맥락상 안맞는 것 같아서
순위가 낮은 h태그나 strong을 쓰는 게 어떨까요?! 

근데 h태그를 쓰자니 페이지마다 우선순위가 달라질 것 같아서 저는 strong태그로 하긴 했음다!

juhyuny commented on src/components/molecules/ReviewCard.tsx:
위 h2 내용과 같습니다!!

soohyn committed:
[Refact] 컴포넌트 제목 태그 수정

- h2 -> strong

Resolves 장소 카드, 리뷰 카드 상세 페이지 이동 기능 추가 #26

juhyuny reviewed (approved):
확인했습니다~!
